kubetest2-tf/deployer/deployer.go

```diff
@@ -142,6 +159,7 @@ func New(opts types.Options) (types.Deployer, *pflag.FlagSet) {
 	}
 	klog.InitFlags(nil)
 	flagSet.AddGoFlagSet(goflag.CommandLine)
+	flagSet.StringVar(&d.TargetProvider, "target-provider", "powervs", "The provider to use (vpc or powervs)")
```

Comment:
this is not required if you use `desc` as above

Reply:
instead add an entry here https://github.com/mkumatag/kubetest2-plugins/blob/df90d8a23d4dadc5fa18cc18ba3e191f181472e4/kubetest2-tf/deployer/deployer.go#L137 to set the default as follows

```diff
	       SetKubeconfig:    true,
               TargetProvider: "powervs"
```

Reply:
Implemented the change.